cg/meta/demultiplex/demux_post_processing.py

```diff
@@ -68,22 +68,26 @@ def set_dry_run(self, dry_run: bool) -> None:
         if dry_run:
             self.demux_api.set_dry_run(dry_run=dry_run)
 
-    def copy_sample_sheet(self) -> None:
-        """Copy the sample sheet from run dir to demux dir"""
-        sample_sheet_path = get_file_in_directory(
-            self.demux_api.run_dir, DemultiplexingDirsAndFiles.SAMPLE_SHEET_FILE_NAME
-        )
-        target_sample_sheet_path = Path(
-            self.demux_api.out_dir, DemultiplexingDirsAndFiles.SAMPLE_SHEET_FILE_NAME
-        )
-        LOG.info(
-            f"Copy sample sheet {sample_sheet_path} from flow cell to demuxed result dir {target_sample_sheet_path}"
+    def copy_sample_sheet(
+        self, sample_sheet_source_directory: Path, sample_sheet_destination_directory: Path
+    ) -> None:
+        """Copy the sample sheet from the flow-cell-run dir to demultiplex-runs dir for a flow cell."""
+        sample_sheet_source: Path = Path(
+            sample_sheet_source_directory, DemultiplexingDirsAndFiles.SAMPLE_SHEET_FILE_NAME
         )
-        shutil.copy(
-            sample_sheet_path.as_posix(),
-            target_sample_sheet_path.as_posix(),
+        sample_sheet_destination: Path = Path(
+            sample_sheet_destination_directory, DemultiplexingDirsAndFiles.SAMPLE_SHEET_FILE_NAME
         )
 
+        if not sample_sheet_destination.exists():
+            LOG.debug(
+                f"Copy sample sheet {sample_sheet_source} from flow cell to demuxed result dir {sample_sheet_destination}"
```

Comment:
Might want to change the log message to the destination directory instead
```suggestion
                f"Copy sample sheet {sample_sheet_source} from flow cell to demuxed result dir {sample_sheet_destination_directory}"
```

Reply:
Yes, sounds good!